CHANGES.rst

```diff
@@ -23,6 +23,9 @@ astropy.coordinates
 - Adds a direct transformation from ``ICRS`` to ``AltAz`` frames. This
   provides a modest speedup of approximately 10 percent. [#11079]
 
+- Adds new ``WGS84GeodeticRepresentation``, ``WGS72GeodeticRepresentation``,
+  and ``GRS80GeodeticRepresentation``. [#11086]
```

Comment:
Should we note what these are good for? E.g., "These are mostly for use inside ``EarthLocation`` but can also be used to convert between geocentric (cartesian) and different geodetic representations directly."